src/server/services/wms/qgswmsutils.cpp

```diff
@@ -74,11 +74,16 @@ namespace QgsWms
     {
       return PNG;
     }
-    else if ( format.compare( QLatin1String( "jpg " ), Qt::CaseInsensitive ) == 0  ||
+    else if ( format.compare( QLatin1String( "jpg" ), Qt::CaseInsensitive ) == 0  ||
```

Comment:
@elpaso @rldhont do you know what's this for? I wonder if that's actually ever used. It wasn't functional before (?) and nobody complained.

Reply:
I think that most people use `image/jpeg` (as well as unit tests) so it probably slipped under the radar.

Reply:
I think it's been there for ages, It's not a standard mime type and untested. I believe the idea was just to be tolerant in what we accept.

Reply:
What's your opinion, fix it, leave it or ditch it?
I think `image/jpg` or `jpeg` would be just as likely to be used, rendering this nothing but a random string.

Reply:
fix it, isn't it ?

Reply:
The code is a) broken b) non standard c) almost random and d) very likely unused. Four reasons for removing. I find it hard to find a good one for keeping.

Reply:
What does "fixing" mean though? Simply removing the `' '` at the end of the string? Or harmonizing with the content type and changing `'jpg'` to `'jpeg'`? Or adding both? Isn't any of that breaking the interface, someone could already be sending requests with `jpg ` right now?

If it was really "broken code" I'd be all in for a fix. But I'm having a hard time seeing this as broken code. It looks more like a desperate attempt, pretending to be tolerant (as confirmed by @elpaso above).

Reply:
@m-kuhn I think that in the old times it was more common for clients to use "jpg" "jpeg" with or without "image/", lower or upper case, I have no idea if that's still the situation today, I guess it's not.

So, if fixing does not cost too much I would leave that possibility, but if we do, we should document and put it under test. 

I don't really have a strong opinion here.